sources/packages/backend/apps/test-db-seeding/src/main.ts

```diff
@@ -0,0 +1,36 @@
+require("../../../env_setup_apps");
+import { NestFactory } from "@nestjs/core";
+import { CleanDb } from "./clean-db/clean-db";
+import { TestDbSeedingModule } from "./test-db-seeding.module";
+import { SeedExecutor } from "./seed-executors/seed-executor";
+// Clean db command line parameter.
+const CLEAN_DB = "--task=clean-db";
+
+async function bootstrap() {
+  const app = await NestFactory.create(TestDbSeedingModule);
+
+  // Checking for CLEAN_DB parameter.
+  if (process.argv.includes(CLEAN_DB) && process.argv.length === 3) {
+    // Clean db.
+    await app.get(CleanDb).onModuleInit();
+    console.info("Database cleaned!!");
+    return;
```

Comment:
I believe that the return here means that both parameters cannot be provided at the same time unless I am missing something. Is it needed? I guess that cleaning the DB and executing specific seeds would be pretty normal, IMO.

Reply:
ya... you are correct. if the `--task=clean-db`executed, then to run the test. we need to do the migrations first. so, both comments should run separately.